### PR TITLE
0.5

### PR DESCRIPTION
**Pull request by nsheff**
Would like to release July 11th.

### CODE REVIEW

File: refgenie/asset_build_packages.py [hunk 1]
```diff
@@ -150,19 +150,21 @@
...
-            "cp {{refgene}} {{asset_outfolder}}/{{genome}}_{annsfile}.format".format(annsfile=GENE_ANNO_FILE_NAME)
+            "cp {{annogene}} {{asset_outfolder}}/{{genome}}_{annsfile}".format(annsfile=GENE_ANNO_FILE_NAME)
         ]
     },
     "TSS_annotation": {
         "required_inputs": [],
         "required_assets": [GENE_ANNO_NAME],
+        #"container": "databio/refgenie",
         "command_list": [
             "zcat {{asset_outfolder}}/{{genome}}_{annsfile} | awk {awk_cmd} | LC_COLLATE=C sort -k1,1 -k2,2n -u > {{asset_outfolder}}{{genome}}_TSS.tsv".
```
Comment (nsheff) [inline refgenie/asset_build_packages.py]: zcat is not cross-platform

File: refgenie/refgenie.py [hunk 2]
```diff
@@ -29,7 +29,7 @@
 INSERT_CMD = "add"
 
 
-BUILD_SPECIFIC_ARGS = ('fasta', 'gtf', 'context')
+BUILD_SPECIFIC_ARGS = ('fasta', 'gtf', 'context', 'annogene')
```
Comment (nsheff) [inline refgenie/refgenie.py]: what is the annogene exactly? is it not the same as the gtf file?

File: refgenie/asset_build_packages.py (hunk 1 above)
Comment (vreuter) [reply refgenie/asset_build_packages.py]: I copied what's in pepatac; what should be substituted?

File: refgenie/refgenie.py (hunk 2 above)
Comment (vreuter) [reply refgenie/refgenie.py]: refGene.txt.gz --> refGene.txt

Comment (vreuter) [reply refgenie/refgenie.py]: https://github.com/databio/pepatac/blob/dev/docs/faq.md#how-do-i-calculate-tss-enrichments

File: refgenie/asset_build_packages.py (hunk 1 above)
Comment (vreuter) [reply refgenie/asset_build_packages.py]: https://github.com/databio/pepatac/blob/dev/docs/faq.md#how-do-i-calculate-tss-enrichments

**Pull request merged**